modules/typeable/src/main/scala/shapeless3/typeable/typeable.scala

```diff
@@ -412,7 +436,7 @@ trait TypeCase[T] extends Serializable {
 }
 
 object TypeCase {
-  import syntax.typeable.given
+  import syntax.typeable.*
```

Comment:
Wait, does this also import `given`?

Reply:
This seems to be necessary to import the extension.